### PR TITLE
Add environment variable support for server

### PR DESCRIPTION
**Pull request by BHRnR**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,8 +18,9 @@ Home Brewing Automation Software
 * Run `$ yarn server` to start development server
 * Run `$ yarn client` to start development client
 
-## Data Path
+## .env file
 
-Define source of data in app.ts
+* Copy & rename .env.example to .env in same path
```
Comment (scip92) [inline README.md:23]: You can use \`{content}\` to hightlight code

File: server/src/app.ts [hunk 2]
```diff
@@ -18,13 +20,14 @@ function readCurrentTemperature(): number {
     }
 }
 
+console.log(process.env.PATH_SAMPLE_DATA)
+console.log("Test")
```
Comment (scip92) [inline server/src/app.ts:24]: Do we still need this console.logs?

File: server/src/app.ts [hunk 3]
```diff
@@ -1,12 +1,14 @@
...
 import { noCors } from "./no-cors";
+import dotenv from 'dotenv';
+dotenv.config();
 
 var fs = require('fs');
 const app = express();
-const filePath = "C:/Users/Benjamin Hacker/Desktop/Programmieren/temperature"
-const getRandomTemperature = true
-const refreshTime = 1
+const filePath = process.env.PATH_SAMPLE_DATA
```
Comment (scip92) [inline server/src/app.ts:9]: I would prefer `const sensorPath = process.env.SENSOR_PATH`

File: server/src/app.ts [hunk 4]
```diff
@@ -1,12 +1,14 @@
...
+const getRandomTemperature = process.env.USE_FAKER
+const refreshTime = process.env.MEASUREMENT_INTERVAL
```
Comment (scip92) [inline server/src/app.ts:11]: I would prefer `const measurementInterval = process.env.MEASUREMENT_INTERVAL`

File: server/.env.example [hunk 5]
```diff
@@ -0,0 +1,4 @@
+PORT=5000
+SENSOR_PATH="{project_dir}/{sensor_data}"
+MEASUREMENT_INTERVAL=1
+USE_FAKER=true
```
Comment (scip92) [inline server/.env.example:4]: Do you use this config flag already?

File: server/src/app.ts [hunk 6]
```diff
@@ -18,13 +20,14 @@ function readCurrentTemperature(): number {
     }
 }
 
+console.log(process.env.PATH_SAMPLE_DATA)
+console.log("Test")
+
 setInterval(() => {
-    console.log("eine Sekunde")
     temperatures.push(readCurrentTemperature())
-    console.log(temperatures)
```
Comment (scip92) [inline server/src/app.ts:24]: Nice cleanup!

File: server/src/app.ts (hunk 2 above)
Comment (BHRnR) [reply server/src/app.ts:24]: Removed.

File: server/src/app.ts (hunk 3 above)
Comment (BHRnR) [reply server/src/app.ts:9]: Done.

File: server/src/app.ts (hunk 4 above)
Comment (BHRnR) [reply server/src/app.ts:11]: Done

File: server/.env.example (hunk 5 above)
Comment (BHRnR) [reply server/.env.example:4]: Yes. (readCurrentTemperature true case)

**Pull request merged**